Automattic/wc-calypso-bridge
# Manually bundle gridicons

We can't pull in gridicons recursively, and it is also missing a `composer.json`. 
See https://getcomposer.org/doc/faqs/why-can%27t-composer-load-repositories-recursively.md

For internal launch, let's bundle the function manually, with a TODO for cleaning this up.

To Test:
* Apply and view a page like the setup task list, to confirm gridicons/checkmarks display.
